src/components/ImageGeneration/GeneratedImageActions.tsx

```diff
@@ -61,7 +63,7 @@ export function GeneratedImageActions({
   const hasSelected = !!selectedCount;
 
   return (
-    <div className="flex items-center justify-between gap-6">
+    <div className={`flex items-center justify-between ${isMobile ? '' : 'gap-6'}`}>
```

Comment:
We don't want to use `isMobile` here. Tailwind has its own way of targeting different screen sizes. In general, try to avoid using `isMobile` for anything that could affect layout. When changing screen sizes, this can lead to some pretty bad layout jank.

Reply:
I would suggested targeting smallest screen sizes as base then using tailwind breakpoints to adjust to bigger screen sizes. For example
```suggestion
    <div className="flex items-center justify-between gap-6 md:gap-0">
```